src/components/Label/Label.tsx

```diff
@@ -34,6 +35,10 @@ export interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement>
      * visual appearance of the label
      */
     emphasis?: "strong" | "normal";
+    /**
```

Comment:
Please rename the property to `additionalElements` and use "Add other elements to the end of the label content".